src/editor/layout/index.js

```diff
@@ -34,14 +34,16 @@ export default compose( [
 	const blockPreview = content ? parse( content ) : null;
 	return (
 		<Fragment>
-			<div className="block-editor-patterns newspack-newsletters__layout-panel-preview">
-				<div className="block-editor-patterns__item">
-					<div className="block-editor-patterns__item-preview">
-						<BlockPreview blocks={ blockPreview } viewportWidth={ 568 } />
+			{ templateId > 0 && (
```

Comment:
Nitpick: not making the assumption about `templateId`'s meaning seems more understandable to me
```suggestion
			{ blockPreview !== null && (
```

Reply:
Good idea. Less "implementation-specific." https://github.com/Automattic/newspack-newsletters/pull/130/commits/d03352e6bd1d3d71469e524c6fcccf45d0bd406d